### PR TITLE
accelerators: always show toast message when adjusting controls

### PR DESCRIPTION
**Pull request by elstoc**
As discussed in #6688, this changes the keyboard accelerators so that slider and combobox adjustments always show a toast message regardless of whether the module being adjusted is visible.

### CODE REVIEW

File: src/gui/accelerators.c [hunk 1]
```diff
@@ -617,26 +614,23 @@ void dt_accel_widget_toast(GtkWidget *widget)
...
+        dt_toast_log(_("%s: %s"), w->label, text);
+    }
+    else
+    { //label is empty
+      if(w->module && w->module->multi_name[0] != '\0')
+        dt_toast_log(_("%s %s / %s"), w->module->name(), w->module->multi_name, text);
+      else if(w->module)
+        dt_toast_log(_("%s / %s"), w->module->name(), text);
+      else
+        dt_toast_log(_("%s"), text);
```
Comment (TurboGit) [inline src/gui/accelerators.c:633]: we don't need the `_(...)` here as there is nothing to translate Above it is different as there is the `/` or `:` that could be changed or spaced differently.

**Review by TurboGit — CHANGES_REQUESTED** [review]
A minor translation to remove.

**Comment by elstoc** [comment]
done

**Comment by TurboGit** [comment]
Thanks.

**Pull request merged**